### PR TITLE
[huggingface_tensorflow, huggingface_pytorch] update for Transformers to 4.11.0

### PR DESCRIPTION
**Pull request by philschmid**
.. 4.11.0

*GitHub Issue #, if available:*

Note: If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

### Description

This PRs updates the `transformers` and `datasets` versions inside the huggingface PyTorch and TensorFlow DLC.

### DLC image/dockerfile

_Change only in `buildspec.yaml` required._

## Label Checklist
- [ ] I have added the project label for this PR (*<project_name>* or "Improvement")

## PR Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [x] If the PR changes affects SM test, I've modified dlc_developer_config.toml in my PR branch by setting sagemaker_tests = true and efa_tests = true
- [x] If this PR changes existing code, the change fully backward compatible with pre-existing code. (Non backward-compatible changes need special approval.)
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Pytest Marker Checklist
- [ ] (If applicable) I have added the marker `@pytest.mark.model("<model-type>")` to the new tests which I have added, to specify the Deep Learning model that is used in the test (use `"N/A"` if the test doesn't use a model)
- [ ] (If applicable) I have added the marker `@pytest.mark.integration("<feature-being-tested>")` to the new tests which I have added, to specify the feature that will be tested
- [ ] (If applicable) I have added the marker `@pytest.mark.multinode(<integer-num-nodes>)` to the new tests which I have added, to specify the number of nodes used on a multi-node test
- [ ] (If applicable) I have added the marker `@pytest.mark.processor(<"cpu"/"gpu"/"eia"/"neuron">)` to the new tests which I have added, if a test is specifically applicable to only one processor type

#### EIA/NEURON Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `ei_mode = true` or `neuron_mode = true`

#### Benchmark Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `benchmark_mode = true`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: dlc_developer_config.toml [hunk 1]
```diff
@@ -25,19 +25,19 @@ do_build = true
 
 [test]
 sanity_tests = true
-  safety_check_test = false
+safety_check_test = false
```
Comment (arjkesh) [inline dlc_developer_config.toml:28]: did you do one commit with `safety_check_test` = true?

File: dlc_developer_config.toml [hunk 2]
```diff
@@ -25,19 +25,19 @@ do_build = true
 
 [test]
 sanity_tests = true
-  safety_check_test = false
+safety_check_test = true
```
Comment (arjkesh) [inline dlc_developer_config.toml:28]: when you revert, can you add the two spaces back? it's not functional, just shows that safety check is a subset of sanity

**Comment by arjkesh** [comment]
@philschmid - looks like the safety checks have passed. Can you revert the TOML and we'll get this merged?

**Review by arjkesh — APPROVED** [review]
